PYCS_Language/Operations/IfElseOperationHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace SplitAndMerge
+{
+    class IfElseOperationHandler
+    {
+        public void IfElseExecutor(string value, VariablesHandler varsHandler)
+        {
+            ConsoleHandler consoleHandler = new ConsoleHandler();
+            value = value.Trim();
+
+            if (value.Contains(BasicOperationsConstants.LET))
+            {
+                varsHandler.VaribleOperation(value);
+            }
+            else if (value.Contains(BasicOperationsConstants.CONSOLE))
+            {
+                consoleHandler.PrintToConsole(value, varsHandler.Variables);
+            }
+            else
+            {
+                Console.WriteLine("Incorrect sintax");
```

Comment:
U use this string "Incorrect sintax" very often, you can create a separate class with error messages and use to avoid a lot of repeat code
And sintax -> syntax

Reply:
I want to replace this Console's in next task, they are temporary